Quadr_Eq.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include<cmath>	
+#include<iostream>
+using namespace std;
+
+constexpr size_t two{ 2 }, four{ 4 };// ��������� ����� 2 � 4(�� �������, �� ���� ��� ���� ������, �� ������� � �������� � ������ calculation
+
+class Quadr_Eq
+{
+	double a{}, b{}, c{}; // ����������� �, B � ����� �
+	mutable double* result = nullptr;//��������� �� ����������(mutable ������������ ��� ����, ���� ����� ����������� �������� ���������� ���� � ����������� �������...
+	mutable double Discriminant=0;// ... �� ���������� ��� ���� ����, ������� ���������������� ����� �����������

```

Comment:
рекомендую использовать список инициализации
```suggestion
	mutable double Discriminant{};

```